vector/src/main/java/im/vector/app/features/analytics/screen/ScreenEvent.kt

```diff
@@ -25,8 +24,6 @@ import timber.log.Timber
  * Track a screen display. Unique usage.
  */
 class ScreenEvent(val screenName: MobileScreen.ScreenName) {
```

Comment:
The class `ScreenEvent`has been created to measure the time the screen is displayed. Now that we do not do that anymore, I would vote to just remove it.